modules/update-upgrading-oc-adm-upgrade-status.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * updating/updating_a_cluster/updating-cluster-cli.adoc
+
+:_mod-docs-content-type: CONTEXT
+[id="update-upgrading-oc-adm-upgrade-status_{context}"]
+= Using `oc adm upgrade status` to retrieve information about a cluster update (Technology Preview)
+
+When updating your cluster, it is useful to understand how your update is progressing. While the `oc adm upgrade` command returns limited information about the status of your update, this release introduces the `oc adm upgrade status` command as a Technology Preview feature. This command decouples status information from the `oc adm upgrade` command and provides specific information regarding a cluster update, including the status of the control plane and worker node updates.
+
+The `oc adm upgrade status` command only returns information on the status of your current cluster update. It does not provide read or write capabilities and will not alter any information in your Operator. 
```

Comment:
"It does not provide read or write capabilities" sounds weird. Any command does not provide read/write capabilities. The command *reads* the state of the cluster but never writes or modifies anything. That's probably what the second half of the sentence says, but IMO "information in your Operator" is not precise, I'd say something like "never alters any state in your cluster" (I guess "operator" slipped in by mistake).